samples/HostConfig/sampleFontStyles.json

```diff
@@ -0,0 +1,155 @@
+{
+	"spacing": {
+		"small": 3,
+		"default": 8,
+		"medium": 20,
+		"large": 30,
+		"extraLarge": 40,
+		"padding": 15
+	},
+	"separator": {
+		"lineThickness": 1,
```

Comment:
These should not be new hostConfigs but we should have the current host configs have the fonts options.